dolphinscheduler-ui/src/js/conf/home/pages/datasource/pages/list/_source/createDataSource.vue

```diff
@@ -21,7 +21,7 @@
         <m-list-box-f>
           <template slot="name"><strong>*</strong>{{$t('Datasource')}}</template>
           <template slot="content" size="small">
-              <el-select style="width: 100%;" v-model="type">
+              <el-select style="width: 100%;" v-model="type" :disabled="this.item.id ? true:false">
```

Comment:
:disabled="this.item.id"
Is this modification feasible?

Reply:
Thanks @zhuangchong , done this.